homeassistant/components/litterrobot/sensor.py

```diff
@@ -179,7 +180,19 @@ class RobotSensorEntityDescription(SensorEntityDescription, Generic[_WhiskerEnti
         native_unit_of_measurement=UnitOfMass.POUNDS,
         state_class=SensorStateClass.MEASUREMENT,
         value_fn=lambda pet: pet.weight,
-    )
+    ),
+    RobotSensorEntityDescription[Pet](
+        key="visits_today",
+        translation_key="visits_today",
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
why DIAGNOSTIC? Since it is not info about the litter robot, I am not sure if it should be in the diagnostic category.